OreshkoJA/lab_2/LAB2.cpp

```diff
@@ -0,0 +1,63 @@
+#include <iostream>
+#include <cstdlib>
+#include <ctime> 
+using namespace std;
+const int con = 10;
+int main()
+{
+	int j = 0;
+	int i, s;
+	cout << "Number 1:";
+	srand(time(0));
+	{int arr[con];
+	for (i = 0; i < con; i++)
+	{
+		arr[i] = rand();
+		cout << arr[i] << ' ';
+	}
+	cout << endl;
+i = con;
+while (i > 0) {
+	j = i;
+	while (arr[j] > arr[j - 1]) {
+		s = arr[j];
+		arr[j] = arr[j - 1];
+		arr[j - 1] = s;
+		j++;
+	}
+	i--;
+}
+		cout << "Number 2:";
+		for (i = 0; i < con; i++)
+		{
+			cout << arr[i] << ' ';
+		}
+		cout << endl;
+		int min = arr[0];
+		int max = arr[0];
+		double sum = 0;
+		cout << "Odd numbers:";
+		for (i = 0; i < con; i++) {
+			if (arr[i] % 2) { cout << arr[i] << " "; }

```

Comment:
You have to make new array using odd values from first array, not just print them